content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -49,7 +49,10 @@ Instructions for other distributions or Kubernetes installations may vary.
 
 ## Create a static pod {#static-pod-creation}
 
-You can configure a static Pod two different ways: either by using configuration file(s) or by HTTP.
+You can configure a static Pod in two different ways: either by using configuration file(s) or by HTTP.
```

Comment:
I'd prefer to see inline links rather than separate bullet points. 
```suggestion
You can configure a static Pod with either a [file system hosted configuration file](/docs/tasks/configure-pod-container/static-pod/#configuration-files) or a [web hosted configuration file](/docs/tasks/configure-pod-container/static-pod/#pods-created-via-http).
```